README.md

```diff
@@ -58,11 +58,11 @@ Run Examples
 ```
 Hadoop (verified with Hadoop 2.8.1 on Ubuntu 16.04)
```

Comment:
I notice on PR #9 that Hadoop 3.0 was used.   Can we stick to 2.8 for now?

Reply:
yes, will revert it to 2.8.3